src/styles/index.scss

```diff
@@ -1,3 +1,12 @@
-body {
-  margin: 0;
-}
+@import './variables';
+@import './page';
+@import './product-card-block';
+@import './reviews-stars-block';
+
+
+
+
+
+
+
+
```

Comment:
Add only one blank line to the end of file
<img width="563" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/53841847-72c7-41ef-8224-d868302cb4fd">
```suggestion
@import './reviews-stars-block';

```